add-ons/dynamic-admission-controllers/dynamic-admission-controllers.addon

```diff
@@ -0,0 +1,18 @@
+# Name: dynamic-admission-controllers
+# Description: Enables ValidatingAdmissionWebhook & MutatingAdmissionWebhook
+# Url: https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/dynamic_admission_controllers.html
+# OpenShift-Version: >=3.10.0
+
+echo Enalble required addmission configs
```

Comment:
Typo: Enable

Reply:
Thank you for checking this.
I've fixed documentations including this.